WalletWasabi/Tor/TorProcessManager.cs

```diff
@@ -140,6 +141,31 @@ private async Task RestartingLoopAsync(CancellationToken globalCancellationToken
 					{
 						throw new NotSupportedException(TorProcessStartedByDifferentUser, ex);
 					}
+
+					// Check if the bridges in the running Tor instance are the same as user requested.
+					TorControlReply clientTransportPluginReply = await controlClient.GetConfAsync(keyword: "ClientTransportPlugin", cancellationToken).ConfigureAwait(false);
+					if (!clientTransportPluginReply.Success)
+					{
+						throw new InvalidOperationException("Tor control failed to report the current transport plugin.");
+					}
```

Comment:
Its the same comment as L152, it should be about transport plugins.
Also, shouldn't them be checked too for accuracy below?